ci-operator/step-registry/ipi/gcp/ovn/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- abhinavdahiya
```

Comment:
I’d suggest having more than one approver from whoever is going go to own the jobs that will use this workflow. 

Why just Abhinav?